sources/common/infoschema.go

```diff
@@ -120,8 +120,8 @@ func ProcessData(conv *internal.Conv, infoSchema InfoSchema) {
 				srcSchema.Name, ok))
 			continue
 		}
-		// Extract spColds without synthetic primary key columnn id.
-		colIds := RemoveSynthId(conv, tableId, spSchema.ColIds)
+		// Extract common spColds.
+		colIds := GetCommonColumnIds(conv, tableId, spSchema.ColIds)
```

Comment:
For my understanding, can you explain what's happening here? 
(Maybe also add a comment on why this is needed)

Reply:
added